airflow/www/templates/airflow/dag.html

```diff
@@ -317,12 +317,12 @@ <h4>Task Actions</h4>
             </div>
           </form>
           <hr style="margin-bottom: 8px;">
-          <form method="POST" data-action="{{ url_for('Airflow.success') }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
```

Comment:
Why did we remove this? I'm running the branch locally and getting a `Bad Request The CSRF token is missing.` error when marking a dag run as success or failure. Task instances work fine.